lib/web/apiserver.go

```diff
@@ -2691,16 +2691,23 @@ func (h *Handler) siteNodeConnect(
 	h.log.Debugf("New terminal request for server=%s, login=%s, sid=%s, websid=%s.",
 		req.Server, req.Login, sessionData.ID, sessionCtx.GetSessionID())
 
-	authAccessPoint, err := site.CachingAccessPoint()
-	if err != nil {
-		h.log.WithError(err).Debug("Unable to get auth access point.")
-		return nil, trace.Wrap(err)
-	}
+	keepAliveInterval := req.KeepAliveInterval
+	// Try to use the keep alive interval from the request.
+	// When it's not set or below a second, use the cluster's keep alive interval.
+	if keepAliveInterval < time.Second {
+		authAccessPoint, err := site.CachingAccessPoint()
+		if err != nil {
+			h.log.WithError(err).Debug("Unable to get auth access point.")
```

Comment:
I'm not a fan of using `WithError` at debug level.

If it's informational only, like this message is, we probably don't need the stack trace that `WithError` generates. Stack traces in logs generally attract attention, especially when troubleshooting.

```suggestion
			h.log.Debugf("Unable to get auth access point: %v", err)
```